mozilla/fx-private-relay
# Add "tips" card at the bottom of the dashboard

# New feature description

Add "tips" card at the bottom of the dashboard

It currently holds a single tip, namely how to create a custom
alias, that's only visible when the user has created a subdomain
but doesn't have any custom aliases yet.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/150532171-acfe3d7b-220d-4530-9ea9-121dbbdc5c6a.png)

# How to test

Log in with an account that:

- Has Premium
- Has a subdomain set
- Does not have any custom aliases yet

You should then see it at the bottom of the screen. Also make sure that it disappears if you hit the "X", and stays gone after you refresh. (At least as long as you are logged in with the same user.)

# Checklist

- [ ] All acceptance criteria are met. - They also say it should overlay the cards, but I've asked whether we can not do that, because it's obnoxious :)
